ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -7,6 +7,100 @@
   },
   timelineFile: 'e6s.txt',
   triggers: [
+    {
+      id: 'E6S Strike Spark',
+      regex: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexDe: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexFr: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4BD3', capture: false }),
+      regexJa: Regexes.startsUsing({ source: ['イフリート', 'ラクタパクシャ'], id: '4BD3', capture: false }),
+      promise: function(data) {
+        let p = new Promise(async (res) => {
+          // helper function to delay the promise execution for the given time
```

Comment:
Hmm.  I wonder if delaySeconds should come before promise, so that promises can lean on delaySeconds to do this.

Reply:
That'd be great. All triggers for positional data receiving I wrote this tier, are related to retrieving combatatant data after a delay. Also I assume the usage of promises are fairly new in cactbot, which makes changing it rather non-impactful, or?

Reply:
Yeah, promises *just* got added by @valarnin and there's not any use of them yet other than in a test.  So, very easy to change.  I'll try to do that later tonight.